completion/expl3-commands.cwl

```diff
@@ -0,0 +1,4322 @@
+# expl3 commands, l3kernel 2024-11-02
+# command list generated by https://github.com/James-Yu/LaTeX-Workshop/blob/master/dev/latex3command.py
+# argument specifications added by Matthew Bertucci 2024/12/16
+
+\bitset_addto_named_index:Nn %<⟨bitset var⟩%> {%<⟨name1=index1,name2=index2,...⟩%>}
+\bitset_clear:N %<⟨bitset var⟩%>
+\bitset_clear:c {%<⟨bitset var name⟩%>}
```

Comment:
Maybe we want a consistent use (or dropping) of `⟨...⟩` brackets. Not a blocker.

<details><summary>Stats of uses of <code>⟨...⟩</code></summary>
<p>

```
$ rg -F '⟨' --count
metre.cwl:17
exerquiz.cwl:3
physics2.cwl:12
pdfextra.cwl:1
eqexam.cwl:9
textopo.cwl:9
semantex.cwl:1
ajmacros.cwl:4
tikzlibrarybraids.cwl:2
plaintex.cwl:133
hiero.cwl:2
wargame.cwl:10
plext.cwl:1
expkv-cs.cwl:10
china2e.cwl:1
luatexja-ajmacros.cwl:4
qcircuit.cwl:1
babel.cwl:10
tikzlibraryext.shapes.circlearrow.cwl:1
texmate.cwl:4
ptex.cwl:45
nath.cwl:6
class-elteikthesis.cwl:5
longnamefilelist.cwl:2
tikzlibraryceltic.cwl:1
tikzlibraryext.transformations.mirror.cwl:2
tex.cwl:12
abspos.cwl:2
xskak.cwl:6
namedef.cwl:2
yathesis.cwl:109
mdwtab.cwl:7
class-iopart.cwl:2
prettytok.cwl:3
algxpar.cwl:4
doafter.cwl:1
easybase.cwl:8
optex.cwl:86
```

</p>
</details>

Reply:
I'm a fan of using the angle brackets with "undelimited" arguments. To me `⟨bitset var⟩` signifies "insert something here" that a merely greyed out `bitset var` does not. Since most document commands have delimited arguments, they are not necessary, but I agree consistency would be better. Not sure how easy a mass change would be since I think the brackets need to be inside `%<...%>` which would break a lot of cwl argument types.

Reply:
I agree the unbraced arguments need angle brackets more than braced ones. So, I'm OK to just keep them as is.